src/components/AutoSuggest/AutoSuggest.tsx

```diff
@@ -75,6 +86,10 @@ const AutoSuggest: FC<AutoSuggestProps> = ({
   const clearInput = useCallback(() => {
     if (inputRef.current) {
       inputRef.current.value = ''
+
+      if (!showInlineList) {
+        inputRef.current.focus()
```

Comment:
is it possible to avoid 2 deep nesting here?

Reply:
Not really, because the field should only get the focus when `inputRef` has a value AND `showInlineList` equals to false. I could flatten the assertion like so `if (inputRef.current && !showInlineList)`, but that doesn't add much to the readability.

Reply:
ok